RHDEVDOCS-6306: Release Notes 1.3 (#87048)
openshift/openshift-docs: modules/ob-release-notes-1-3.adoc

// This module is included in the following assemblies:
// * about/ob-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="ob-release-notes-1-3_{context}"]
= Release notes for {builds-shortname} General Availability 1.3

{builds-shortname} General Availability (GA) 1.3 is now available on {ocp-product-title} <version>.

[id="new-features-1-3_{context}"]
== New features

The following section highlight what is new in {builds-shortname} 1.3.

* With this update, {builds-shortname} for OpenShift components, including the Operator, Shipwright-Builds, and the Shared Resource Container Storage Interface (CSI) driver, are FIPS-validated and run on FIPS-compliant clusters.

* With this update, the {builds-operator} deploys Shipwright v0.14 APIs, controllers and webhook.

> * With this update, while creating the `build` or `buildRun` resources using OpenShift Builds Client, you can set the parameters using the --param-value flag.

Reviewer: Can we highlight `--param-value`, since it is a command snippet?

[id="fixed-issues-1-3_{context}"]
== Fixed issues

The following section highlights fixed issues in {builds-shortname} 1.3.

* Before this update, when you installed {builds-shortname} without installing {pipelines-shortname}, the {builds-shortname} attempts to deploy {pipelines-shortname} and enables the Tekton Pipeline controllers. Previously, {builds-shortname} incorrectly configured {pipelines-shortname} and failed to deploy the Tekton Pipeline controllers. With this change, {builds-shortname} creates a valid TektonConfig and deploys the Tekton Pipeline controllers.

> * Before this update, when you tried to install {builds-shortname} on ARM, IBM Power, or IBM Z worker node CPU architectures, then the build strategies would fail to execute the build strategies image. The build strategies image was available only for AMD architecture. Now, the AMD only image is replaced with a multi-arch image in `buildah` and `source-to-image` build strategies and you can install {builds-shortname} on all the architectures.

Reviewer: ```suggestion
* Before this update, when you tried to execute builds using {builds-shortname} installed on ARM, IBM Power, or IBM Z worker node CPU architectures, then the builds would fail due to AMD only build strategies images. Now, the AMD only image is replaced with a multi-arch image in `buildah` and `source-to-image` build strategies and you can execute builds on all the architectures.
```
